components/atom/tag/src/Actionable/Container.js

```diff
@@ -18,6 +26,16 @@ ActionableTagContainer.propTypes = {
   Link: PropTypes.func,
   href: PropTypes.string,
   target: PropTypes.oneOf(['_self', '_blank', '_parent', '_top']),
+  rel: PropTypes.arrayOf(
+    PropTypes.oneOf([
```

Comment:
replace by: `PropTypes.oneOf(Object.values(LINK_TYPES))` so there is no need to modify this line if you add more LINK_TYPES in the future.